corehq/apps/es/es_query.py

```diff
@@ -139,11 +134,11 @@ class ESQuery(object):
     }
 
     def __init__(self, index=None, for_export=False):
-        if index is not None:
-            self.index = index
+        if index is None:
+            index = self.index  # use class attribute
```

Comment:
Doing it this way means that if index is specifed and different from `self.index`, then outside of this method, `self.index` and `self.adapter` will disagree.  That sounds like a potential source of confusion.

Reply:
Good catch. I'm pretty sure there isn't any live code where this actually happens. I'll make a note to revisit this.

Reply:
thanks!